### ISSUE TITLE
place a legend for the branches on the project description panel 

### ISSUE THREAD

**Issue by gissoo** [Princeton-CDH/lenape-timetree#137]

**Comment by gissoo** [comment]
@rlskoeser @jhimpele [Please take a look here on Figma](https://www.figma.com/file/sGOn9MZy6ML0yNxuiQJ8u5/Lenape-Time-Tree?node-id=1268%3A2092&t=FSaqKRHIk4L15NMU-1) and let me know what you think about the way the legend is structured on desktop and mobile

**Comment by gissoo** [comment]
@jhimpele I thought we discussed putting two in a line to save vertical space so we wouldn't have to scroll to see the entire legend, it would be displayed without the need to scroll. What do you think?

**Comment by jhimpele** [comment]
@gissoo You're right...I remember now that we did. But it is not in a line (i.e. row) on the example. I see two columns. Either way the sequence let to right might help link the two, especially since the greens look very similar in the legend.

**Comment by gissoo** [comment]
@jhimpele I was thinking it would be read from left to right rather than column 1, 2. We could probably add supporting text above it that reads "branches on the tree are positioned in the order below (left-right)" ..or something like this.

**Comment by jhimpele** [comment]
What do you think @rlskoeser ? I tend to think that columns are read down and from left to right...Let's avoid having to write out the position sequence!   Might another option be one single column?

**Comment by rlskoeser** [comment]
The order bothers me a bit as well — I can't tell if it is order exactly or the way the colors go (or both), but I think I would rather have them go down in columns than left to right across columns. (At least, I'd like to see that option.)

I don't think we have to write anything about the order, they should be able to see that by comparing the legend to the tree now that we have the branch sequence and grouping working better.

**Comment by gissoo** [comment]
@rlskoeser @jhimpele [how do you feel about this?](https://www.figma.com/file/sGOn9MZy6ML0yNxuiQJ8u5/Lenape-Time-Tree?node-id=1355%3A2169&t=iaSL01uLrmEeK9OM-1) Good point about reading! I should have thought about it while we were discussing it at the meeting.

**Comment by rlskoeser** [comment]
This looks ok and is certainly more readable, but I think it will be pretty hard to get the intro content that brief. Are you ok with the list scrolling past the visible part of the intro details?

**Comment by gissoo** [comment]
@rlskoeser @jhimpele also [please check this one out ](https://www.figma.com/file/sGOn9MZy6ML0yNxuiQJ8u5/Lenape-Time-Tree?node-id=1355%3A2706&t=iaSL01uLrmEeK9OM-1)

**Comment by rlskoeser** [comment]
That column order and color sequence looks good to me if we want to stick with the more compact column layout.

**Comment by jhimpele** [comment]
I could go either way on these two versions. I have shortened the text for that panel, and have an idea for making it even shorter that I can describe when we see each other.....

**Comment by gissoo** [comment]
@rlskoeser @jhimpele thank you! I applied your new description. In either case we'll have to do a bit of scrolling, [here is the one with revised columns](https://www.figma.com/file/sGOn9MZy6ML0yNxuiQJ8u5/Lenape-Time-Tree?node-id=1355%3A2706&t=iaSL01uLrmEeK9OM-1), and [here is the one with the simple list format](https://www.figma.com/file/sGOn9MZy6ML0yNxuiQJ8u5/Lenape-Time-Tree?node-id=1355%3A2169&t=iaSL01uLrmEeK9OM-1)

I recommend going with the revised columns to minimize the scroll.

**Comment by jhimpele** [comment]
I prefer the double columns because it doesn’t leave too much empty space. I will re-write a better opening phrase/sentence…..


Jeffrey Himpele.
Director, VizE Lab for Ethnographic Data Visualization
Lecturer in Anthropology
vizelab.princeton.edu<http://vizelab.princeton.edu/>


From: gissoo ***@***.***>
Date: Monday, April 10, 2023 at 4:25 PM
To: Princeton-CDH/lenape-timetree ***@***.***>
Cc: Jeffrey D. Himpele ***@***.***>, Mention ***@***.***>
Subject: Re: [Princeton-CDH/lenape-timetree] place a legend for the branches on the project description panel (Issue #137)

@rlskoeser<https://github.com/rlskoeser> @jhimpele<https://github.com/jhimpele> thank you! I applied your new description. In either case we'll have to do a bit of scrolling, here is the one with revised columns<https://www.figma.com/file/sGOn9MZy6ML0yNxuiQJ8u5/Lenape-Time-Tree?node-id=1355%3A2706&t=iaSL01uLrmEeK9OM-1>, and here is the one with the simple list format<https://www.figma.com/file/sGOn9MZy6ML0yNxuiQJ8u5/Lenape-Time-Tree?node-id=1355%3A2169&t=iaSL01uLrmEeK9OM-1>

I recommend going with the revised columns to minimize the scroll.
